portfolio/portfolio.go

```diff
@@ -261,16 +258,16 @@ func (p *Base) GetPersonalPortfolio() map[string]float64 {
 // getPercentage returns the percentage of the target coin amount against the
 // total coin amount.
 func getPercentage(input map[string]float64, target string, totals map[string]float64) float64 {
```

Comment:
RM `map[string]float64` type dec. for input var.

Reply:
Gonna leave this one as is since the target is of type string

Reply:
How about
`func getPercentage(target, totals map[string]float64, target string) float64` 
?

Reply:
Out of scope for this PR, there's probabably a lot of others like this but is insignificant. Malalignment checks (mainly for structs) can be done in the future.